invenio_cli/cli.py

```diff
@@ -107,68 +53,51 @@ def init(flavour):
-              help='Which environment to build, it defaults to local')
 @click.option('--pre', default=False, is_flag=True,
               help='If specified, allows the installation of alpha releases')
 @click.option('--lock/--skip-lock', default=True, is_flag=True,
               help='Lock dependencies or avoid this step')
-def build(pre, local, lock):
-    """Either setups project locally or builds container images.
+def install(pre, lock):
+    """Installs the  project locally.
 
-    --local option installs dependencies, creates instance directory,
+    Installs dependencies, creates instance directory,
     links invenio.cfg + templates, copies images and other statics and finally
     builds front-end assets.
-
-    --containers creates docker images inside of which the equivalent of
-    --local has been run.
     """
     cli_config = CLIConfig()
-    commands = Commands(cli_config, local)
-    commands.build(pre=pre, lock=lock)
+    commands = LocalCommands(cli_config)
+    commands.install(pre=pre, lock=lock)
 
 
 @cli.command()
-@click.option('--local/--containers', default=True, is_flag=True,
-              help='Which environment to build, it defaults to local')
 @click.option('--force', default=False, is_flag=True,
               help='Force recreation of db tables, ES indices, queues...')
 def services(local, force):
     """Starts services and ensures they are setup (DB, ES, queue, etc.).
 
-    TODO: Forward any extra argument to docker-compose.
+    --force destroys and resets services
     """
     cli_config = CLIConfig()
     commands = Commands(cli_config, local)
     commands.services(force=force)
 
 
-# TODO: Remove when --containers implemented in the above
 @cli.command()
-@click.option('--local/--containers', default=True, is_flag=True,
-              help='Which environment to build, it defaults to local')
 @click.option('--force', default=False, is_flag=True,
-              help='Delete all content from the database, ES indexes, queues')
-@click.option('--stop-containers', default=False, is_flag=True, required=False,
-              help='Stop containers when finishing the setup operations.')
-@click.option('--verbose', default=False, is_flag=True, required=False,
-              help='Verbose mode will show all logs in the console.')
-def setup(local, force, stop_containers, verbose):
-    """Sets up the application for the first time (DB, ES, queue, etc.)."""
-    # Create config object
-    invenio_cli = InvenioCli(verbose=verbose)
-
-    click.secho('Setting up environment for {flavour} application...'
-                .format(flavour=invenio_cli.flavour), fg='green')
-
-    # Initialize docker client
-    docker_helper = DockerHelper(local=local,
-                                 log_config=invenio_cli.log_config)
+              help='Force recreation of db tables, ES indices, queues...')
+@click.option('--no-install-js', default=False, is_flag=True,
```

Comment:
minor: same `skip` vs `no...`

Reply:
See discussion above. I am fine with revisiting once others try it out.